sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -93,45 +65,26 @@
  * @see CertificateClientBuilder
  * @see PagedFlux
  */
-@ServiceClient(builder = CertificateClientBuilder.class, isAsync = true, serviceInterfaces = CertificateService.class)
+@ServiceClient(builder = CertificateClientBuilder.class, isAsync = true, serviceInterfaces = CertificateClientImpl.CertificateService.class)
```

Comment:
CertificateClientImpl is in the implementation folder. Does it allow to be explored?

Reply:
nvm